packages/playground/src/data/mockOverviewRealData.ts

```diff
@@ -17,8 +23,8 @@ export const mockOverviewRealData = [
         pullreq_id: 59,
         order: 2,
         sub_order: 0,
-        type: 'branch-delete',
-        kind: 'system',
+        type: 'branch-delete' as EnumPullReqActivityType,
```

Comment:
Can we add type info to `mockOverviewRealData` instead so that this type cast is not required?